src/TropicalGeometry/linear_space.jl

```diff
@@ -260,20 +273,17 @@ Min tropical linear space
 
 ```
 """
-function tropical_linear_space(A::MatElem, nu::Union{Nothing,TropicalSemiringMap}=nothing; weighted_polyhedral_complex_only::Bool=false)
-    # if nu unspecified, initialize as the trivial valuation + min convention
-    isnothing(nu) && (nu=tropical_semiring_map(base_ring(A)))
+function tropical_linear_space(A::Union{MatElem,Matrix}, nu::Union{Nothing,TropicalSemiringMap}=nothing; weighted_polyhedral_complex_only::Bool=false)
+    # convert to Oscar matrix (necessary for AbstractAlgebra.minors)
+    A isa Matrix && (A = matrix(parent(first(A)),A))
```

Comment:
Optionally:
```suggestion
tropical_linear_space(A::AbstractMatrix, nu::Union{Nothing,TropicalSemiringMap}=nothing; weighted_polyhedral_complex_only::Bool=false) =
    tropical_linear_space(matrix(parent(first(A)),A), nu; weighted_polyhedral_complex_only)
function tropical_linear_space(A::MatElem, nu::Union{Nothing,TropicalSemiringMap}=nothing; weighted_polyhedral_complex_only::Bool=false)
```
Just make sure to attach the docs to the right function, so probably the top function needs to be moved below.

Reply:
I like your version better, I'll change it!